sdk/python/packages/flet-core/src/flet_core/video.py

```diff
@@ -178,6 +187,9 @@ def before_update(self):
         if isinstance(self.__subtitle_configuration, VideoSubtitleConfiguration):
             self._set_attr_json("subtitleConfiguration", self.__subtitle_configuration)
 
+        if isinstance(self.configuration, VideoConfiguration):
```

Comment:
I'd use `self.__configuration` here instead of `self.configuration` like everywhere else.

Reply:
Nice catch! 👍🏾